tests/PythonTests.cmake

```diff
@@ -114,21 +101,16 @@ function(add_python_test case)
     set(name ${name}.${fn_SUBMODULE})
   endif()
 
-  if(PYTHON_COVERAGE)
-    add_test(
-      NAME ${name}
-      WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
-      COMMAND "${PYTHON_COVERAGE_EXECUTABLE}" run --parallel-mode "--rcfile=${PYTHON_COVERAGE_CONFIG}"
-              "--source=girder,${PROJECT_SOURCE_DIR}/clients/python/girder_client${other_covg}"
-              -m unittest -v ${module}
-    )
-  else()
-    add_test(
-      NAME ${name}
-      WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
-      COMMAND "${PYTHON_EXECUTABLE}" -m unittest -v ${module}
-    )
-  endif()
+  file(MAKE_DIRECTORY "${PROJECT_SOURCE_DIR}/build/test/coverage/python_temp")
+
+  add_test(
+    NAME ${name}
+    WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}"
+    COMMAND "${PYTHON_COVERAGE_EXECUTABLE}" run
```

Comment:
Do we need the `--parallel-mode` flag to make distinctly named coverage file?

Reply:
This option is now hardcoded into the `.coveragerc` file.